### PR TITLE
implement rudimentary explorer experience

### PR DESCRIPTION
**Pull request by pandaro**
A small contribution, it may be accepted, if it is consistent with the rest of the project.
- It stimulates the exploration of the map, as close to a newly generated chunck player receives a small experience point.
- Surely it is from rebalancing.
- I can write better code.
But first I want to know if my contribution is welcome.

### CODE REVIEW

**Comment by cdqwertz** [comment]
:+1: I really like the idea of encourage exploring. 
Could you move the code to the xp mod or skills mod?

File: mods/explorer_xp/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+License for Code
+----------------
+
+Copyright (C) 2016 cd2 (cdqwertz) <padarogames@gmail.com>
```
Comment (cdqwertz) [inline mods/explorer_xp/LICENSE.txt]: Please fix this line.

**Comment by pandaro** [comment]
yes I can move it where you think is appropriate.
skill or xp? for me is better in xp
you agree?
as an aside, have you a irc or something similar? i need to ask you some things about your gameplay ideas

**Comment by cdqwertz** [comment]
Ok, xp is fine.

Unfortunately, I cannot join any irc channel at the moment, but you can
just open an issue to ask me anything.

**Comment by cdqwertz** [comment]
Looks good 👍

**Comment by pandaro** [comment]
when you think is time feel free to merge it

**Comment by cdqwertz** [comment]
Done

**Pull request merged**